Add missing meta descriptions (#235)
mongodb/docs-php-library: source/reference/method/MongoDBCollection-createSearchIndexes.txt

MongoDB\\Collection::createSearchIndexes()
==========================================

.. meta::

>    :description: Create Atlas Search or Vector Search indexes for a collection using `createSearchIndexes()` in PHP, specifying index definitions and options.

Reviewer: ```suggestion
   :description: Create Atlas Search or Vector Search indexes for a collection with the MongoDB PHP Library, specifying index definitions and options.
```
